### PR TITLE
Adjust wait time for cloud-transition test failures

### PR DESCRIPTION
**Pull request by soumyakoduri**
Adjust wait time for cloud-transition tests

Signed-off-by: Soumya Koduri <skoduri@redhat.com>

### CODE REVIEW

**Comment by soumyakoduri** [comment]
http://pulpito.front.sepia.ceph.com/soumyakoduri-2022-04-25_06:23:06-rgw:cloud-transition-wip-skoduri-dbstore-tests-distro-basic-smithi

File: s3tests_boto3/functional/test_s3.py [hunk 1]
```diff
@@ -10443,20 +10443,21 @@ def test_lifecycle_noncur_cloud_transition():
...
-    time.sleep(5*lc_interval)
+    time.sleep(7*lc_interval)
     expire1_keys = list_bucket_storage_class(client, bucket)
     eq(len(expire1_keys['STANDARD']), 2)
     eq(len(expire1_keys[sc[1]]), 0)
 
     if (retain_head_object == None or retain_head_object == "false"):
         eq(len(expire1_keys[sc[2]]), 0)
     else:
+        print(sc[2])
```
Comment (alimaredia) [inline s3tests_boto3/functional/test_s3.py:10460]: Is this `print` statement necessary?

Comment (soumyakoduri) [reply s3tests_boto3/functional/test_s3.py]: ah no. It was for debugging. Fixed it.

**Comment by soumyakoduri** [comment]
> @soumyakoduri if you're happy with this, feel free to merge and cherry-pick to the branches that need it

sure..thanks! 
With the latest changes (https://github.com/ceph/ceph/pull/46008/files) to run tests on different cloudtier config params, the tests have been taking lot more time. So I am still adjusting the wait time. Will merge the patch once the results have been consistent.

**Comment by soumyakoduri** [comment]
http://pulpito.front.sepia.ceph.com/soumyakoduri-2022-05-09_10:52:51-rgw:cloud-transition-wip-skoduri-dbstore-vstart-distro-basic-smithi

**Comment by soumyakoduri** [comment]
Cherry-picked this patch to ceph-quincy & ceph-master as well.

**Pull request merged**